docs/manuals/user/dataMigrationFromSth-cometToQL/README.md

```diff
@@ -0,0 +1,46 @@
+**To migrate data from STH-Comet to QuantumLeap**
+
+Fiware GE STH-Comet support NGSIv1 while QuantumLeap support NGSIv2. As NGSIv1 is being deprecated from Fiware Orion so using QuantumLeap will be preferred.
+If we need to store data that was previously in STH-Comet to QuantumLeap we will have to follow the following steps:
+
+1. Exporting Data from MongoDB
+2. Adding Data into CrateDB
+
+Migration-0.0.1-SNAPSHOT.jar file can be used to perform above task automatically. 
+
+Jar file have dependency over ```application.properties``` , ```convert_json.py``` and ```dependency-jars``` to present at the same location as that of jar.
+
+To run the jar user will have to configure following parameters in application.properties file:
```

Comment:
The docs so far implies some prior-knowledge about what the `application.properties` file is. Some people might not even work with Java :). You don't need to explain everything here, but I'd suggest adding something like: "this tool is based on language X framework Y, and these are the links to the info you need to know in order to use this tool"